retailer-oas/retailer.yaml

```diff
@@ -251,21 +301,116 @@ paths:
       responses:
         '204':
           description: No Content
-        '400':
+        '404':
           description: Invalid ID
-  
+
+  /picking-location/{id}/schedule:
+    post:
+      summary: 'Creates a Picking Location Schedule'
```

Comment:
Nit: Should we normalize entity type names? We have a mix of "Picking Location" and "picking location". I wouldn't mind capitalizing it to denote importance and make it consistent.